src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -291,7 +291,7 @@ public void Write(EventLevel level, Exception innerException, string message, pa
             if (!HeaderWritten)
             {
                 // Obtain the current library version dynamically.
-                WriteAlways(string.Format(CultureInfo.InvariantCulture, _versionLogMessage, typeof(IdentityModelEventSource).GetTypeInfo().Assembly.GetName().Version.ToString()));
+                WriteAlways(string.Format(CultureInfo.InvariantCulture, _versionLogMessage, typeof(IdentityModelEventSource).GetTypeInfo().Assembly!.GetName().Version!.ToString()));
```

Comment:
I don't understand, why is Assembly! and Version! required here?

Reply:
I'm not sure `!` is needed for either. Do you get warnings without?

Reply:
@jennyf19  Yeap, without it, IDE throws error code CS8602-Dereference of a possibly null reference. Version is not nullable in System.Reflection so it causes noise

Reply:
I had Build on, not Build + IntelliSense, so I didn't see this one. You can actually do `GetTypeInfo().Assembly.GetName().Version?.ToString()));` instead